Common/src/Storage/SessionData.cs

```diff
@@ -32,15 +33,18 @@ public record SessionData(string        SessionId,
                           SessionStatus Status,
                           DateTime      CreationDate,
                           DateTime?     CancellationDate,
+                          IList<string> PartitionIds,
```

Comment:
Why using a Ilist here and at some other place IEnumerable?

Reply:
I think it should even be `string[]`
@wkirschenmann opinion?

Reply:
kept IList